.github/workflows/ci.yml

```diff
@@ -213,9 +213,19 @@ jobs:
       - uses: dsherret/rust-toolchain-file@v1
       - run: echo ::add-matcher::.github/workflows/rust_matcher.json
 
+      # Make sure the `Cargo.lock` file reflects the latest available versions.
+      # This is what users would end up with on a fresh module, so we want to
+      # catch any compile errors arising from a different transitive closure
+      # of dependencies than what is in the workspace lock file.
+      #
+      # For context see also: https://github.com/clockworklabs/SpacetimeDB/pull/2714
+      - name: Update dependencies
+        run: cargo update
+
       - name: Build module-test
         run: cargo run -p spacetimedb-cli -- build --project-path modules/module-test
 
+      # TODO(kim): What does this have to do with bindgen?
```

Comment:
Should we remove this step? Seems redundant

Reply:
Ah - it should be changed to instead run the tests from `spacetimedb-codegen`, since the codegen tests were moved from cli to that crate.